stdlib/public/Platform/CMakeLists.txt

```diff
@@ -119,12 +121,16 @@ foreach(sdk ${SWIFT_SDKS})
       list(APPEND glibc_modulemap_target_list ${copy_glibc_modulemap_static})
     endif()
 
-    set(glibc_header_out "${module_dir}/SwiftGlibc.h")
-    handle_gyb_source_single(glibc_header_target
-        SOURCE "SwiftGlibc.h.gyb"
-        OUTPUT "${glibc_header_out}"
-        FLAGS "-DCMAKE_SDK=${sdk}")
-    list(APPEND glibc_modulemap_target_list ${glibc_header_target})
+    # FreeBSD uses a different module map that does not use this header,
+    # so we don't generate it
```

Comment:
Have you tried using this "Glibc" header instead on FreeBSD? I have it almost working for Android, #35707, so I think it should work for the BSDs too.

Reply:
I tried, but ran into a lot of missing functions. I did not have time to figure out what the problem is, yet, but I will take a look. Thanks for the link. I would like to land this first, though.